### DISCUSSION TITLE
Is there any difference between a generic FA2010 and core of the NuXT/Micro8088 images?

### DISCUSSION THREAD

**Discussion by TzOk83** [640-KB/GLaBIOS#34] [Tech Support] [answered]
I've got a question like in topic - is there any difference between a generic FA2010 and core of the NuXT/Micro8088 images? I understand, that image for Micro 8088 contains some companion BIOS modules, but is the core BIOS the same? I'd like to compose my own set of ROMs for my Micro 8088, without XT-IDE, with different HD-FDD BIOS, and with BASIC. So should I bother to extract the core image from the the dedicated NuXT/Micro 8088 image, or can I just use a generic FT2010A V20 image?

**Comment by 640-KB** [comment]
Great question.  Aside from the additional companion ROMs added there are a few differences in the core BIOS for that build:

1. Boots to 9.54 MHz, 2 RAM WS compared to default FE2010A which boots to safer 4 RAM WS
2. DRAM refresh is disabled since Micro8088/NuXT use SRAM (this is a performance boost)
3. RAM Parity checking is disabled since it's not used on that platform
4. CGA snow removal is disabled by default (it's only needed for early generation CGA adapters)
5. Storage units are displayed in IEC "MiB, GiB" rather than "traditional" BIOS units "MB, GB" (consistency with micro_8088 BIOS)

If you're curious, this is located [here in the source code](https://github.com/640-KB/GLaBIOS/blob/05e253cbe6bbf8230ba0c203e9adf5877a204501/src/GLABIOS.ASM#L313).

Note: the 8K 0.2.4 ROMs are now in the Releases section but the bundled builds haven't been updated yet.

**Comment by TzOk83** [comment]
Ok, thanks for the quick answer. If the core BIOS is special for Micro8088, can a "raw" 8kB Micro8088 binary build be included in the releases?

**Comment by 640-KB** [comment, marked as answer]
Absolutely.  I just uploaded the raw 8K BIOS images to the release section.  I had just held off because I did want to personally test the fully built ROMs on my own NuXT but I don't suspect there will be any issues since the changes between 0.2.3 and 0.2.4 are fairly minor.  Will follow up with full images once I can give them a full check out though!
